### PR TITLE
Automate Deviant Abilities

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

**Comment by TikaelSol** [comment]
Consume energy's effect will need a change to ChoiceSet to allow for choosing any strike. We need a way to handle the backlash tracking as well.

File: packs/data/classfeatures.db/strained-metabolism.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            "value": null
+        },
+        "description": {
+            "value": "<p>Your body's overclocked physiology requires larger-than- usual amounts of nourishment, leaving you weakened when it kicks in mid-combat.</p>\n<p><strong>Mild Backlash</strong> Your metabolism immediately burns through your body's energy stores, making you @UUID[Compendium.pf2e.conditionitems.Enfeebled]{Enfeebled 1} and @UUID[Compendium.pf2e.conditionitems.Clumsy]{Clumsy 1} until you take a bite of food (which usually requires 2 Interact actions: one to retrieve the food and another to consume it).</p>\n<p><strong>Moderate Backlash</strong> Ravenous hunger wracks your body. You are enfeebled 1, clumsy 1, and @UUID[Compendium.pf2e.conditionitems.Fatigued]{Fatigued} until you can eat a full meal, which requires at least 10 minutes.</p>\n<p><strong>Severe Backlash</strong> The strain on your body is truly incapacitating. You become @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 2} as nausea wracks you, and you are also enfeebled 1 and clumsy 1 until you can eat a full meal (which requires you to no longer be sickened). You are fatigued until you can get a full night's rest.</p>"
```
Comment (simonward) [inline packs/data/classfeatures.db/strained-metabolism.json:13]: Extra space after hyphen "larger-than- usual",

File: packs/data/classfeatures.db/wraith-deviant-classification.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
+{
+    "_id": "8Jjy7VsQAJJqCeyE",
+    "img": "systems/pf2e/icons/spells/adapt-self.webp",
+    "name": "Wraith  Deviant Classification",
```
Comment (simonward) [inline packs/data/classfeatures.db/wraith-deviant-classification.json:4]: Extra space after Wraith.

File: packs/data/feat-effects.db/effect-consume-energy-augment-strike.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{
+    "_id": "eMsI1lR0SuJBCYjn",
+    "img": "systems/pf2e/icons/spells/divinity-leech.webp",
+    "name": "Effect: Consume Energy (Augment Strike)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Strikes with the chosen weapon or unarmed attack deal an additional 1d6 damage of a type matching the energy you consumed</p>"
```
Comment (simonward) [inline packs/data/feat-effects.db/effect-consume-energy-augment-strike.json:8]: Missing period.

File: packs/data/feats.db/distant-wandering.json [hunk 4]
```diff
@@ -21,7 +21,17 @@
...
-        "rules": [],
+        "rules": [
+            {
+                "key": "GrantItem",
+                "predicate": [
+                    {
+                        "not": "feature:wraith-deviant-classification"
+                    }
+                ],
+                "uuid": "Compendium.pf2e.classfeatures.Wraith  Deviant Classification"
```
Comment (simonward) [inline packs/data/feats.db/distant-wandering.json:32]: Extra space in this and other references.

**Review by simonward — CHANGES_REQUESTED** [review]

**Pull request merged**